src/contexts/StringContext.tsx

```diff
@@ -58,6 +62,10 @@ export const defaultStrings: WizardStrings = {
     syncButtonTooltip: 'Refresh',
     required: 'Required',
     expandToFixValidationErrors: 'Expand to fix validation errors',
+    selectNoItems: 'Select none (0 items)',
+    selectPageItems: 'Select page',
+    selectAllItems: 'Select all',
+    itemsString: 'items',
```

Comment:
I suggest that you remove `itemsString` and change `selectPageItems` and `selectAllItems` into functions that the wizard can pass a count into. The function we provide can use our `t` function. So the string would be `"Select page ({{count}} items)"` for example. We should never really concatatenate strings from different languages because we are assuming the grammar rules when we do that.